doc/classes/Plane.xml

```diff
@@ -147,36 +140,36 @@
 			<argument index="0" name="point" type="Vector3">
 			</argument>
 			<description>
-				Returns the orthogonal projection of point [code]p[/code] into a point in the plane.
+				Returns the orthogonal projection of [code]point[/code] into a point in the plane.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="d" type="float" setter="" getter="" default="0.0">
-			Distance from the origin to the plane, in the direction of [member normal].
+			The distance from the origin to the plane, in the direction of [member normal]. This value is typically non-negative.
```

Comment:
```suggestion
			The distance from the origin to the plane, in the direction of [member normal]. This value is typically non-negative.
			In the scalar equation of the plane [code]ax + by + cz = d[/code], this is [code]d[/code], while the [code](a, b, c)[/code] coordinates are represented by the [member normal] property.
```

Reply:
Note: Sometimes the equation is written as `ax + by + cz + d = 0`, in which case `d` would be the opposite of the one described in my suggestion. I didn't check which case we use in Godot.

Reply:
Seems to be the one you suggested, since for plane `(0, 1, 0, 1)`, the point `(0, 1, 0)` is on it, `0*0 + 1*1 + 0*0 = 1`.